python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -86,8 +86,9 @@ def __init__(self, timeout=10, n_parallel=None, build_func='default'):
                 build_func = ndk.create_shared
             else:
                 raise ValueError("Invalid build_func" + build_func)
-
-        self.build_func = _wrap_build_func(build_func)
+            self.build_func = _wrap_build_func(build_func)
```

Comment:
The original code is correct. You should change your build_func instead.

Reply:
in this case we have custom build_func for vta that we want to call to directly: https://github.com/tmoreau89/tvm/blob/dev/vta/python/vta/build_module.py#L123-L177

Reply:
actually I've addressed this by getting rid of the `vta_autotvm_build_func`; and instead now we're using the common build infrastructure